src/app/AppNotFoundView.vue

```diff
@@ -1,30 +1,37 @@
 <template>
-  <div class="overview">
-    <KEmptyState>
-      <template #title>
-        <WarningIcon class="mb-3" />
+  <RouteView
+    name="app-not-found-view"
+  >
+    <AppView>
+      <div class="overview">
+        <KEmptyState>
+          <template #title>
+            <WarningIcon class="mb-3" />
+            <h1>
+              <RouteTitle
+                title="Page Not Found"
+                :render="true"
+              />
+            </h1>
+          </template>
 
-        <p>Page Not Found</p>
-      </template>
+          <template #message>
+            <p>The page or entity you were looking for does not exist.</p>
+          </template>
 
-      <template #message>
-        The page or entity you were looking for does not exist.
-      </template>
-
-      <template #cta>
-        <KButton
-          appearance="primary"
-          :to="{ name: 'home' }"
-        >
-          Go Home
-        </KButton>
-      </template>
-    </KEmptyState>
-  </div>
+          <template #cta>
+            <KButton
+              appearance="primary"
+              :to="{ name: 'home' }"
+            >
+              Go Home
+            </KButton>
+          </template>
+        </KEmptyState>
+      </div>
+    </AppView>
+  </RouteView>
```

Comment:
I noticed out Page Not Found view was missing RouteView/Title and AppView, kinda not related to this PR, but as I was there I added it here.